### DISCUSSION TITLE
Is it possible to have a new field called section, able to customize the way it is layout.

### DISCUSSION THREAD

**Discussion by taoguangc** [statamic/cms#6157] [General]
It can create a container, similar to raw in bootstrap, it can specify the internal layout.
Just like some visual editor, e.g. elementor.
![image](https://user-images.githubusercontent.com/31331096/171976881-e78114df-684a-4e8e-b973-40123dffc7fc.png)

**Comment by edalzell** [comment]
Yes there are a few examples in the marketplace,  like this one: https://statamic.com/addons/rias/position-fieldtype
